requirements.txt

```diff
@@ -1,5 +1,4 @@
 requests
 haversine
 httplib2
--e git://github.com/pchampin/hydra-py.git#egg=hydra
--e git+https://github.com/HTTP-APIs/hydrus.git#egg=hydrus
+-e git://github.com/HTTP-APIs/hydra-python-core.git#egg=hydra_python_core
```

Comment:
We don't need the `-e` flag. Please substitute these lines by using the new approach as displayed [here](https://github.com/HTTP-APIs/hydrus/blob/develop/requirements.txt#L26)